src/styles/cards.css

```diff
@@ -0,0 +1,80 @@
+.cards {
+  box-sizing: border-box;
+  color: #060b35;
+  width: 200px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  border: 1px solid #f3f3f3;
+
+  &__foto {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin-top: 4px;
+    margin-bottom: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+
+    color: #616070;
+  }
+  &__reviews {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+  &__price {
+    color: #616070;
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    line-height: 18px;
+    &-text {
+      font-size: 12px;
+      font-weight: 400;
+    }
+    &-price {
+      color: #060b35;
+      font-size: 16px;
+      font-weight: 700;
+    }
+  }
+  &__button {
+    box-sizing: border-box;
+    color: #fff;
+    text-transform: uppercase;
+    background-color: #00acdc;
+    border-radius: 5px;
+    justify-content: center;
+    align-items: center;
+    width: 166px;
+    height: 40px;
+    margin-top: 16px;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    text-decoration: none;
+    display: flex;
+    border: 1px solid #00acdc;
+
+    &:hover {
+      color: #00acdc;
+      background-color: #fff;
+    }
```

Comment:
The BEM element class `.cards__button` and its pseudo-class `:hover` should be written without the `&`.